src/Message/AbstractPart.php

```diff
@@ -19,14 +19,14 @@
     protected $resource;
 
     /**
-     * @var null|string
+     * @var array
      */
-    protected $partNumber;
+    protected $parts = [];
 
     /**
-     * @var array
+     * @var null|string
```

Comment:
`$partNumber` is requested to be a valid string by both object constructor and interface, the docblock can declare it to be a `string` only, you can remove the `null`